GVRf/Framework/src/org/gearvrf/accessibility/GVRAccessibilityManager.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2015 Samsung Electronics Co., LTD
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+package org.gearvrf.accessibility;
+
+import java.util.ArrayList;
+
+import org.gearvrf.GVRContext;
+
+final class GVRAccessibilityManager {
+
+    private ArrayList<GVRAccessibilityTalkBack> mTalkBacks;
+    private GVRAccessibilityZoom mZoom;
+    private GVRAccessibilitySpeech mSpeech;
+    private GVRAccessibilityInvertedColors mInvertedColors;
+
+    public GVRAccessibilityManager(GVRContext gvrContext) {
```

Comment:
For consistency's sake I believe this manager should be obtained from the context as pretty much everything else. The GVRContext has too many things, true, but that is a different topic.